Builderで409エラー

builderなどのテストで以下の様なエラーとなる場合がある。

```
{IsFatal:true, Serial:"xxx", Status:"409 Conflict", ErrorCode:"disconnect_b4_delete", ErrorMessage:"It does not allow the requested operation. Please run after you disconnect the connection to the server."}
```

確認範囲:

- データベース
- ロードバランサ

データベースアプライアンスで手動操作がなくても409エラーとなる場合がある。

```
=== CONT  TestBuilder_Build
    TestBuilder_Build: testing.go:317: Delete is failed:  Error in response: &sacloud.APIErrorResponse{IsFatal:true, Serial:"xxx", Status:"409 Conflict", ErrorCode:"server_power_must_be_down", ErrorMessage:"It does not allow the requested operation. Server does not allow this operation during startup."}
    TestBuilder_Build: testing.go:216: Cleanup is failed: %!(EXTRA *sacloud.apiError=Error in response: &sacloud.APIErrorResponse{IsFatal:true, Serial:"xxx", Status:"409 Conflict", ErrorCode:"disconnect_b4_delete", ErrorMessage:"It does not allow the requested operation. Please run after you disconnect the connection to the server."})
--- FAIL: TestBuilder_Build (231.66s)
FAIL
FAIL	github.com/sacloud/libsacloud/v2/utils/builder/database	231.691s
```

--- a/utils/builder/database/builder_test.go
+++ b/utils/builder/database/builder_test.go
@@ -18,6 +18,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/sacloud/libsacloud/v2/utils/power"
+
 	"github.com/sacloud/libsacloud/v2/utils/builder"
 
 	"github.com/sacloud/libsacloud/v2/sacloud"
@@ -113,6 +115,10 @@ func TestBuilder_Build(t *testing.T) {
 				dbOp := sacloud.NewDatabaseOp(caller)
 				return dbOp.Delete(ctx, testZone, ctx.ID)
 			},
+		},
+		Shutdown: func(ctx *testutil.CRUDTestContext, caller sacloud.APICaller) error {
+			dbOp := sacloud.NewDatabaseOp(caller)
+			return power.ShutdownDatabase(ctx, dbOp, testZone, ctx.ID, true)
 		},
 		Cleanup: func(ctx *testutil.CRUDTestContext, caller sacloud.APICaller) error {
 			swOp := sacloud.NewSwitchOp(caller)

